lib/classes/Variables.js

```diff
@@ -308,7 +308,13 @@ class Variables {
     return BbPromise.all(populations).then((results) =>
       results.forEach((result) => {
         if (result.value !== result.populated) {
-          _.set(target, result.path, result.populated);
+          // We could not use direct `_.set` as due to security measures, it did not allow setting values
+          // if `prototype` was a part of the path and that caused an infinite loop. In our use-case, there are no
+          // security vulnerabilities.
```

Comment:
I think we can get rid of that comment

At least it's not a real `_.set` use case (which is more about ensuring that nested objects on they way exist).

Even if `_.set` would work ok, I think I'd prefer current approach (as `_.set` confuses what we really deal with here)

Reply:
removed :+1: